ptcg_cockatrice.xml

```diff
@@ -2382,8 +2381,8 @@ Draw a card.</text>
 			<manacost>1{U/B}</manacost>
 			<type>Instant</type>
 			<tablerow>3</tablerow>
-			<text>Split second
-Until end of turn, target Pokémon can't attack or block, and its activated abilities can't be activated.</text>
+			<text>Counter target spell. If that spell is countered this way, put it into its owner's hand instead of into that player's graveyard.
```

Comment:
```suggestion
			<text>Counter target spell. If that spell is countered this way, put it into its owner’s hand instead of into that player’s graveyard. Spells with the same name as that spell can’t be cast until your next turn.
```